lab-michael/README.md

```diff
@@ -0,0 +1,30 @@
+# Bitmap Transformer
+
+## Configuration 
+
+* **.gitignore** - contains a file 
+* **.eslintrc** - contains the course linter configuratoin
+* **.eslintignore** - contains the course linter ignore configuration
+* **package.json** - contains npm package config
+  * create a `lint` script for running eslint
+  * create a `test` script for running tests
+* **lib/** - contains module definitions
+* **\_\_test\_\_/asset/** - contains bitmaps for testing
+* **\_\_test\_\_/** - contains unit tests
+
+## Functionality
+A bitmap (`.bmp`) transformer CLI. 
+It reads a bitmap in from disk, runs e color or raster transforms and then writes it out to a new file. This project requires the use of node buffers in order to manipulate binary data.
+
+## Testing 
+* Use BDD `describe` and `test` methods to define discriptive tests and increase readablity
```

Comment:
You don't want to include the lab instructions in your README - since they are instructions for how you should structure your own tests, rather you only want to include information on how a user can run your own code